schema/schema.sql

```diff
@@ -0,0 +1,66 @@
+DROP TABLE IF EXISTS purchase;
+DROP TABLE IF EXISTS item_genre_assignment;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS item;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS format;
+DROP TABLE IF EXISTS country;
+DROP TABLE IF EXISTS subscriber;
+
+
+CREATE TABLE subscriber (
+    subscriber_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    name VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE country (
+    country_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(60) UNIQUE NOT NULL
+);
+
+CREATE TABLE format (
+    format_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(20) UNIQUE NOT NULL
+);
+
+CREATE TABLE artist (
+    artist_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
I feel like it is possible for artists to have the same name? maybe? will have to check this

Reply:
Yeah, I've noticed that some artists can have the same name so I'm removing the unique.